api-reference/connect.md

```diff
@@ -6,20 +6,20 @@ This file documents the main exports of the library.
 
 Connects and returns an `Organization` for `location`.
 
-| Name | Type | Description |
-| :--- | :--- | :--- |
-| `location` | `String` | The Ethereum address or ENS domain of an Aragon organization. |
-| `connector` | `Connector \| [String, Object] \| String` | Accepts a `Connector` instance, and either a string or a tuple for embedded connectors and their config. |
-| `config` | `Object` | The optional configuration object. |
-| `config.readProvider` | `EthereumProvider` | An [EIP-1193](https://eips.ethereum.org/EIPS/eip-1193) compatible object. |
-| `config.chainId` | `Number` | The [Chain ID](https://chainid.network/) to connect to. Defaults to `1`. |
-| returns | `Promise<Organization>` | An `Organization` instance. |
+| Name                  | Type                                      | Description                                                                                              |
+| :-------------------- | :---------------------------------------- | :------------------------------------------------------------------------------------------------------- |
+| `location`            | `String`                                  | The Ethereum address or ENS domain of an Aragon organization.                                            |
+| `connector`           | `Connector \| [String, Object] \| String` | Accepts a `Connector` instance, and either a string or a tuple for embedded connectors and their config. |
+| `config`              | `Object`                                  | The optional configuration object.                                                                       |
+| `config.readProvider` | `EthereumProvider`                        | An [EIP-1193](https://eips.ethereum.org/EIPS/eip-1193) compatible object.                                |
+| `config.chainId`      | `Number`                                  | The [Chain ID](https://chainid.network/) to connect to. Defaults to `1`.                                 |
+| returns               | `Promise<Organization>`                   | An `Organization` instance.                                                                              |
 
 ### Errors
 
-| Type | Description |
-| :--- | :--- |
-| `ConnectionError` | Gets thrown if the connection fails. |
+| Type                   | Description                                            |
+| :--------------------- | :----------------------------------------------------- |
+| `ConnectionError`      | Gets thrown if the connection fails.                   |
```

Comment:
We don't have support for Errors yet. Should we include a quick PR to include them or remove this from the documentation for now?

Reply:
I think we could try to add proper errors (I am thinking in particular about `OrganizationNotFound` which is very common), and we could remove them just before releasing if we are not ready.

Reply:
I agree we should include at least a handler for `OrganizationNotFound `. But let's do it in a different PR.